marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/entities/search/EntitySearchManager.java

```diff
@@ -62,6 +63,18 @@
 
     private static final String MASTERING_AUDIT_COLLECTION_NAME = "mdm-auditing";
     private static final String[] IGNORED_SM_COLLECTION_SUFFIX = {"auditing", "notification"};
+    private static final List<String> ARTIFACT_COLLECTION_NAMES = Arrays.asList(
+            "http://marklogic.com/data-hub/flow",
+            "http://marklogic.com/data-hub/step-definition",
+            "http://marklogic.com/data-hub/steps",
+            "http://marklogic.com/entity-services/models",
+            "http://marklogic.com/data-hub/mappings",
+            "http://marklogic.com/data-hub/steps/mapping",
```

Comment:
We don't need all of these other step collections - we only need "steps".

We do need "mappings" because legacy mappings - which are not in steps - may exist, and they shouldn't show up either.